web/concrete/controllers/backend/page.php

```diff
@@ -38,14 +33,14 @@ public function create($ptID, $parentID = false)
                 if (is_object($parent)) {
                     $d->setPageDraftTargetParentPageID($parent->getCollectionID());
                 }
-                return Redirect::url(\Core::getApplicationURL() . '/' . DISPATCHER_FILENAME . '?cID=' . $d->getCollectionID() . '&ctask=check-out-first&' . Loader::helper('validation/token')->getParameter());
+                return Redirect::url(rtrim($this->app->make('url/canonical'), '/') . '/' . DISPATCHER_FILENAME . '?cID=' . $d->getCollectionID() . '&ctask=check-out-first&' . $this->app->make('helper/validation/token')->getParameter());
             }
         }
     }
 
     public function exitEditMode($cID, $token)
     {
-        if (Loader::helper('validation/token')->validate('', $token)) {
```

Comment:
`helper/validation/token` is fine here, I think we should be using `->make('token')` for the future.